src/pybind/ceph_daemon.py

```diff
@@ -244,7 +244,7 @@ def _load_schema(self):
                 if schema_data.get('nick'):
                     self._stats[section_name][name] = schema_data['nick']
 
-    def run(self, ostr=sys.stdout):
+    def run(self, interval=1, count=None, ostr=sys.stdout):
```

Comment:
`interval` is always passed in from main, right?  So we can probably lose the default =1 here, as it duplicates the default interval=1 in main.